epinorm/norm.py

```diff
@@ -122,6 +129,86 @@ def get_feature_ids(self):
             .dropna()
             .unique()
         )
+    
+    def _admin_levels_table(self):
+        """
+        This method retrives the information in the "countries.csv" and "administrative_units.tsv".
+        It returns a dictionnary that maps a country name (from countries.csv) to a list of its administrative levels.
+        administrative levels are encoded as dictionaries with properties:
+        {
+            name,
+            admin_level,
+            osmId
+        }
+
+        it also tranliterates information from the administrative_units.tsv file.
+        """
+
+        # * create a mapping from country name to its two letter code
+        country_to_code = {}
+        df = pd.read_csv(COUNTRIES_DATA)
+        for _, row in df.iterrows():
+            country_to_code[row["name"]] = row["alpha_2"]
+
+        # add the exceptions
+        country_to_code.update(COUNTRIES_EXCEPTIONS) 
+
+
+        # * now map country code to its admin levels
+        code_to_admin = {}
+        df = pd.read_csv(ADMIN_LEVELS_DATA, sep="\t")
+        for _, row in df.iterrows():
```

Comment:
The `read_table()` method of pandas uses tabs as its default separator.  
And we can also skip creating the intermediate `df` object.

```suggestion
        for _, row in pd.read_table(ADMIN_LEVELS_DATA).iterrows():
```